include/utils/command_structure/drive_commands.h

```diff
@@ -236,17 +294,29 @@ class OdomSetPosition : public AutoCommand {
    * @param odom the odometry system we are setting
    * @param newpos the position we are telling the odometry to take. defaults to (0, 0), angle = 90
    */
-  OdomSetPosition(OdometryBase &odom, const pose_t &newpos = OdometryBase::zero_pos);
+  OdomSetPosition(OdometryBase &odom, const Pose2d &newpos = zero_pos);
 
   /**
    * Run set_position
    * Overrides run from AutoCommand
    * @returns true when execution is complete, false otherwise
    */
   bool run() override;
+  std::string toString(){
+    char headStr[21];
+    sprintf(headStr, "%d", newpos.rotation().degrees());
```

Comment:
same comment as above about meaning of 21. also, i think you want %f here and below